x-pack/plugins/security_solution/public/flyout/index.tsx

```diff
@@ -14,14 +14,19 @@ import type { LeftPanelProps } from './left';
 import { LeftPanel, LeftPanelKey } from './left';
 import { LeftPanelProvider } from './left/context';
 
+// TODO these should be replaced by a more dynamic solution
+//  see https://github.com/elastic/security-team/issues/6247
+export const RIGHT_SECTION_WIDTH = 500;
+export const LEFT_SECTION_WIDTH = 500;
```

Comment:
should this be 1000 instead of 500 for now?

Reply:
ooops, thanks for catching this lol